src/components/input/input-short/InputShort.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import styled from "styled-components";
+import { errorMessageCSS, InputShortComponentCSS } from "./InputShortCSS";
+import { colors } from "../../colors";
+const getDisabledOrErrorStyles = (p) => {
+  if (p.disabled) {
+    return `border-color: ${colors.grey500}; color:${colors.grey500}; &:hover {
+      border-color: ${colors.grey500};
+    }`;
+  }
+  if (p.error) {
+    return `border-color: ${colors.red500};  &:hover {
+      border-color: ${colors.red500};
+    }
+    &:focus {
+      color: white;
+    }
+    `;
+  }
+};
+
+const InputShortComponent = styled.input`
+  ${InputShortComponentCSS};
+  ${(p) => getDisabledOrErrorStyles(p)}
+`;
+
+const ErrorMessage = styled.div`
+  ${errorMessageCSS};
+`;
+
+const InputShort = ({ disabled, error, errorMessage }) => {
+  return (
+    <div>
+      <InputShortComponent
+        id="input-short-component"
+        disabled={disabled}
+        error={error}
+      />
```

Comment:
Are we adding placeholder text? If so, this is missing a placeholder attribute
![image](https://user-images.githubusercontent.com/51865552/187354633-b187043c-0af7-453c-8df8-1a9d8fac66a3.png)

Reply:
added a placeholder prop